### PR TITLE
Disable PAX arm builds temporarily

### PR DESCRIPTION
**Pull request by hemildesai**
Disable PAX ARM builds for now since they are failing currently.

### CODE REVIEW

File: .github/workflows/nightly-rosetta-pax-build.yaml [hunk 1]
```diff
@@ -95,15 +95,25 @@ jobs:
...
-    uses: ./.github/workflows/_build_rosetta.yaml
-    with:
-      ARCHITECTURE: arm64
-      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
-      BASE_LIBRARY: ${{ needs.metadata.outputs.BASE_LIBRARY }}
-      BASE_IMAGE: ${{ needs.metadata.outputs.BASE_IMAGE_ARM64 }}
-    secrets: inherit
+    runs-on: ubuntu-22.04
+    outputs:
+      DOCKER_TAG_MEALKIT: ''
```
Comment (terrykong) [inline .github/workflows/nightly-rosetta-pax-build.yaml:111]: Just for completeness you should add: `DOCKER_TAG_FINAL`

Comment (hemildesai) [reply .github/workflows/nightly-rosetta-pax-build.yaml:111]: Added.

**Review by terrykong — APPROVED** [review]
Assuming presubmit passes, FFTM.

My preference is to maybe open the revert PR after you merge to make sure this doesn't slip thru the cracks

**Comment by yhtang** [comment]
What is the purpose of disabling the failing builds? The PR description did not provide context on the nature, detail, and consequence of the failure. I can guess that this is to avoid the build failure to stop downstream nightly tests, but the PR per se does not provide this information.

@hemildesai Could you please update the PR with the missing information?

**Comment by hemildesai** [comment]
@yhtang Apologies, Added a short description and remediation PR. Let me know if more details are needed.

**Comment by yhtang** [comment]
> @yhtang Apologies, Added a short description and remediation PR. Let me know if more details are needed.

Thanks!

**Pull request merged**